src/App.tsx

```diff
@@ -23,21 +23,20 @@ var todos = (): Promise<{id: string; title: string;}[]> => new Promise((res) =>
 });
 
 function App() {
-  const [state, setState] = React.useState<{ id: string; title: string }[]>([]);
+  const [todoList, setTodoList] = React.useState<{ id: string; title: string }[]>([]);
 
   React.useEffect(() => {
     (async () => {
-      var awaitedTodos = await todos();
-      for (var i = 0; i < awaitedTodos.length; i++) {
-        setState([...state, awaitedTodos[i]]);
-      }
+      const awaitedTodos = await getTodosService();
+      setTodoList(awaitedTodos);
     })()
-  })
```

Comment:
We want to skip applying an effect on each re-renders, so we have to provide which values we want to check. It could be ID or something... in this case, we don't have that value, so we have to add an empty array.